usr/include/platform.h

```diff
@@ -0,0 +1,19 @@
+#if defined(_AIX)
+#include "aix/getopt.h"
+#include "aix/secure_getenv.h"
+#include "aix/endian.h"
+#include "aix/asprintf.h"
+#include "aix/err.h"
+
+#else /* _AIX */
+/* for getopt, getopt_long */
+#include <getopt.h>
+/* for secure_getenv */
```

Comment:
`secure_getenv` does require `#define _GNU_SOURCE`, so shouldn't it be ensured that `_GNU_SOURCE` is defined here?